### DISCUSSION TITLE
How to get online meeting out of draft?

### DISCUSSION THREAD

**Discussion by willmunroe** [code4recovery/12-step-meeting-list#1324] [Q&A] [answered]
I am trying to add a virtual meeting and I can't seem to get it out of draft.

When I hit publish, the form refreshes but it remains in draft. 

I cannot see any error messages, but I do see that the radio button for "Can I currently attend this meeting in person?" keeps reverting back to "Yes"

Has anyone else run into this problem? Anyone found a solution?

Thank you so much!

P.S. This is a great plugin!

**Comment by dvcb** [comment]
@willmunroe For virtual meetings, a Location is still required in order to be listed in the Meeting Guide App. The typical solution is using an approximate location like City, State.

**Reply by willmunroe** [reply]
Thank you, I actually just tried that. I put "Toronto, ON, Canada" in the Location field but there was no change in behaviour.

Thank you!

**Reply by lovethecodelabs** [reply]
Might try not checking in person

**Reply by dvcb** [reply]
Also verify that a URL is provided

**Reply by willmunroe** [reply]
I actually tried both of these. 

I added a URL for the meeting and I checked "no" for "Can I currently attend this meeting in person?"

Is there another checkbox for "in-person" or virtual?

**Reply by dvcb** [reply]
Hmm, can you provide a screenshot of the edit page?

**Reply by lovethecodelabs** [reply]
![Screenshot_20240128_083937_Chrome.jpg](https://github.com/code4recovery/12-step-meeting-list/assets/685087/4d0fa151-61e7-4456-99f0-0cf6b71b859a)

**Reply by lovethecodelabs** [reply]
Oh it is grayed out

**Reply by lovethecodelabs** [reply]
![Screenshot_20240128_084418_Chrome.jpg](https://github.com/code4recovery/12-step-meeting-list/assets/685087/7aee9177-6d9c-441e-8114-862857e3f044)

**Reply by lovethecodelabs** [reply]
I don't see.an end time in your sceenshot

**Reply by willmunroe** [reply, marked as answer]
It seems I figured it out. 

I had put something in "Location" but not address. When I put something in both, I was able to publish.

Thank you so much for your help!

**Reply by lovethecodelabs** [reply]
Was worried the Javascript was getting confused, good.work
